tests/ut/attention/test_attention_mask.py

```diff
@@ -77,80 +86,29 @@ def test_get_splitfuse_attn_mask(self):
         attention_mask_builder = AttentionMaskBuilder(max_seq_len=1024,
                                                       dtype=torch.float16)
         attn_mask = attention_mask_builder.get_splitfuse_attn_mask(
-            seq_lens=[512],
-            query_lens=[512],
-            position=torch.tensor([0]),
+            seq_lens=torch.tensor([10, 20, 100]),
+            position=torch.tensor([7, 8, 9, 18, 19, 99]),
             dtype=torch.float16,
             device=torch.device("cpu"),
         )
-        self.assertEqual(attn_mask.shape, (1, 512))
+        self.assertEqual(attn_mask.shape, (6, 100))
         self.assertEqual(attention_mask_builder._seq_len_cached, 1024)
 
         attn_mask = attention_mask_builder.get_splitfuse_attn_mask(
-            seq_lens=[2048],
-            query_lens=[1024],
-            position=torch.tensor([0]),
+            seq_lens=torch.tensor([10, 3000, 2000]),
+            position=torch.tensor([7, 8, 9, 2999, 1999]),
             dtype=torch.float16,
             device=torch.device("cpu"),
         )
-        self.assertEqual(attn_mask.shape, (1024, 2048))
-
-        attention_mask_builder = AttentionMaskBuilder(max_seq_len=1024,
-                                                      dtype=torch.int8)
-        attn_mask = attention_mask_builder.get_splitfuse_attn_mask(
-            seq_lens=[512],
-            query_lens=[512],
-            position=torch.tensor([0]),
-            dtype=torch.int8,
-            device=torch.device("cpu"),
-        )
-        self.assertEqual(attn_mask.shape, (1, 512))
-
-    def test_use_multiple_masks(self):
-        max_seq_lens = [128, 512, 1024]
-        dtypes = [torch.float16, torch.bfloat16, torch.int8]
-        for max_seq_len, dtype in zip(max_seq_lens, dtypes):
-            with self.subTest(max_seq_len=max_seq_len, dtype=dtype):
-                self._test_use_multiple_masks(max_seq_len, dtype)
-
-    def _test_use_multiple_masks(self, max_seq_len, dtype):
```

Comment:
I don't agree to drop this case in which `AttentionMaskBuilder->get_splitfuse_attn_mask->get_attn_mask` with same `max_seq_len`